### PR TITLE
Move TSC into docs

### PR DESCRIPTION
**Pull request by stichbury**
To resolve https://github.com/kedro-org/kedro/issues/2490

<s>Note: Still waiting on Stephanie's GitHub profile for now.</s>

Moving from the GitHub wiki back into docs (so will update the wiki when this is merged)

Removed Lim, Gabriel and Cvetanka and moved them to past members :-(

### CODE REVIEW

File: docs/source/contribution/technical_steering_committee.md
Comment (yetudada) [inline docs/source/contribution/technical_steering_committee.md:7]: This part will also need to be updated: "We list [Kedro's current and past maintainers](https://github.com/kedro-org/kedro/blob/main/README.md#the-humans-behind-kedro) on GitHub."

File: docs/source/contribution/technical_steering_committee.md
Comment (yetudada) [inline docs/source/contribution/technical_steering_committee.md:100]: We'll also need to update something further down in this doc. There's a section about: "The act of adding or removing maintainers onto the list requires a pull request against [Kedro's README.md file](https://github.com/kedro-org/kedro/blob/main/README.md#the-humans-behind-kedro)." 

This will need to change to creating the PR on this file.

**Review by yetudada — COMMENTED** [review]
Let some minor comments and thank you so much for picking this up!

**Comment by stichbury** [comment]
Now added https://github.com/stephkaiser to the project.

I've copy edited the entire page this time, so it's more accurate. @yetudada I've asked for a re-review -- please could you confirm if Lim should stay as a maintainer or be moved to past team members?

**Comment by stichbury** [comment]
Now added https://github.com/stephkaiser to the project.

I've copy edited the entire page this time, so it's more accurate. @yetudada I've asked for a re-review -- please could you confirm if Lim should stay as a maintainer or be moved to past team members?

File: docs/source/contribution/technical_steering_committee.md
Comment (astrojuanlu) [inline docs/source/contribution/technical_steering_committee.md:3]: Somehow this sentence feels a bit difficult to read but I don't have a better idea 😓

File: docs/source/contribution/technical_steering_committee.md
Comment (astrojuanlu) [inline docs/source/contribution/technical_steering_committee.md:56]: Not for this PR, but maybe it would be cool if we could show pictures too, like https://github.com/readthedocs/readthedocs.org/pull/8441 (not live anymore)

**Review by astrojuanlu — APPROVED** [review]
Approved with two non-blocking comments!

**Comment by yetudada** [comment]
> I'm not entirely sure if Lim should be moved. @yetudada is he still part of the TSC?

I don't think he is. But I guess, we should follow proper process here and get voting sign-off from the team according to: https://docs.kedro.org/en/stable/contribution/technical_steering_committee.html#adding-or-removing-maintainers 

I'll put a 👍🏾 on this.

**Review by yetudada — APPROVED** [review]
I left a minor URL comment but otherwise this looks great! Thank you so much @stichbury. We can await the final votes and then merge if everything is okay.

File: docs/source/contribution/technical_steering_committee.md
Comment (deepyaman) [inline docs/source/contribution/technical_steering_committee.md:53]:
```suggestion
<!-- DO NOT EDIT THIS AND MERGE A PR WITHOUT A VOTE TO SIGN OFF ANY CHANGES -->
```

**Review by deepyaman — APPROVED** [review]
Made a trivial change (extra space), but also approving!

**Pull request merged**
